app/src/main/java/com/ikarabulut/Game.java

```diff
@@ -4,16 +4,20 @@ public class Game {
     private Board board;
     private GameSettings gameSettings;
 
-    public Game(Board theBoard, GameSettings theGameSettings) {
-        board = theBoard;
+    private GameDisplay gameDisplay;
+
+    public Game(Board theBoard, GameSettings theGameSettings, GameDisplay theDisplay) {
```

Comment:
I actually liked the `theBoard` convention lol. I thought it was easier to understand so that when you are passing parameters around you know you are passing _the_Board like the main board being used.

Reply:
I wonder if that was a sticking point for you before?  There is no _the_ Board.  Like it's not a Singleton.  There is _a_ Board, and there may be many of them.  There is a currently active Board that we care about.

Don't know if that mental metaphor is helpful or not.